### DISCUSSION TITLE
OpenAPI importing nothing

### DISCUSSION THREAD

**Discussion by joshmarinacci** [GitbookIO/community#569] [0. General bugs and questions]
I have an OpenAPI 3.0.3 JSON spec file that I want to add to my docs.  I'm using the OpenAPI advanced block. I have uploaded the JSON file without error, but when I select it as the source it says there are zero blocks within it to select.  Is something wrong with my JSON spec file? Is there a place where parsing errors are reported?

**Comment by joshmarinacci** [comment]
Nevermind. I figured out the 'info' section of my file was missing.  It would be helpful if the importer would flag and display errors instead of swallowing them.

**Reply by taszap** [reply]
Hi @joshmarinacci thanks, I will close this discussion but I raised this internally as a feature request - I agree, displaying the error would be helpful!
